src/components/Option.vue

```diff
@@ -1,18 +1,70 @@
 <template>
   <div class="option">
-    <slot name="title"></slot>
-    <slot name="paragraph"></slot>
-    <slot name="list"></slot>
+    <h3 class="Signika-Semibold">{{ title }}</h3>
+    <p class="Signika-Light">{{ description }}</p>
+    <ul class="ingredients">
+      <li class="checkmark" v-for="(ingredient, index) in ingredients" :key="index">{{ ingredient }}</li>
+    </ul>
     <slot name="button"></slot>
   </div>
 </template>
 
 <script>
 export default {
-  name: 'Option'
-}
+  name: "Option",
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    description: {
+      type: String,
+      required: true
+    },
+    ingredients: {
+      type: Array,
+      required: true
+    }
+  }
+};
 </script>
 
-<style scoped>
+<style lang="scss">
```

Comment:
I think its better if the styles in Options.vue are scoped :)

Reply:
Done 😃